src/util/path.cpp

```diff
@@ -58,4 +58,15 @@ to_absolute_path(nonstd::string_view path)
   }
 }
 
+std::string
+to_absolute_path_no_drive(nonstd::string_view path)
```

Comment:
It would be great it you could add some tests of this to `unittest/test_util_path.cpp`.

Reply:
Added.